packages/fields/src/types/Content/Implementation.js

```diff
@@ -110,45 +101,35 @@ export class Content extends Relationship {
     // to this list+field and don't collide.
     const type = `${GQL_TYPE_PREFIX}_${itemQueryName}_${path}`;
 
-    const blocks = Array.isArray(inputBlocks) ? inputBlocks : [];
+    // Normalise blocks to always be a tuple with a config object
+    let blocks = (Array.isArray(inputBlocks) ? inputBlocks : []).map(block =>
+      Array.isArray(block) ? block : [block, {}]
+    );
+
+    blocks.push(...DEFAULT_BLOCKS);
 
-    blocks.push(
-      ...DEFAULT_BLOCKS.filter(
-        defaultBlock =>
-          !blocks.find(
-            block => (Array.isArray(block) ? block[0] : block).type === defaultBlock.type
```

Comment:
Removed this check as there is a duplicate check following. I know the semantics are slightly different, but the end result is the same; there should only ever be a single `paragraph` block, and that's added by default.